core/camel-core-reifier/src/main/java/org/apache/camel/reifier/dataformat/ForyDataFormatReifier.java

```diff
@@ -20,18 +20,18 @@
 
 import org.apache.camel.CamelContext;
 import org.apache.camel.model.DataFormatDefinition;
-import org.apache.camel.model.dataformat.FuryDataFormat;
+import org.apache.camel.model.dataformat.ForyDataFormat;
 
-public class FuryDataFormatReifier extends DataFormatReifier<FuryDataFormat> {
-    public FuryDataFormatReifier(CamelContext camelContext, DataFormatDefinition definition) {
-        super(camelContext, (FuryDataFormat) definition);
+public class ForyDataFormatReifier extends DataFormatReifier<ForyDataFormat> {
+    public ForyDataFormatReifier(CamelContext camelContext, DataFormatDefinition definition) {
+        super(camelContext, (ForyDataFormat) definition);
     }
 
     @Override
     protected void prepareDataFormatConfig(Map<String, Object> properties) {
         properties.put("unmarshalType", definition.getUnmarshalType());
         properties.put("requireClassRegistration", definition.getRequireClassRegistration());
         properties.put("threadSafe", definition.getThreadSafe());
-        properties.put("allowAutoWiredFury", definition.getAllowAutoWiredFury());
+        properties.put("allowAutoWiredFury", definition.getAllowAutoWiredFory());
```

Comment:
@oscerd should this become 'allowAutoWiredFory'? The PR is large but I think that this property was renamed elsewhere in this PR.